clang/test/Driver/sycl-offload-amdgcn.cpp

```diff
@@ -9,6 +9,11 @@
 // RUN: | FileCheck -check-prefix=CHK-ARCH %s
 // CHK-ARCH: error: missing AMDGPU architecture for SYCL offloading; specify it with '-Xsycl-target-backend --offload-arch'
 
+// RUN: not %clangxx -### -std=c++11 -target x86_64-unknown-linux-gnu -fsycl \
+// RUN: -fsycl-targets=spir64,amdgcn-amd-amdhsa %s 2>&1 \
+// RUN: | FileCheck -check-prefix=CHK-MULTI-ARCH %s
+// CHK-MULTI-ARCH: error: missing AMDGPU architecture for SYCL offloading; specify it with '-Xsycl-target-backend=amdgcn-amd-amdhsa --offload-arch'
```

Comment:
Usually, the architecture is specified like this: `-Xsycl-target-backend=amdgcn-amd-amdhsa --offload-arch=gfx908`
Would it be helpful to include the `--offload-arch` value in the error message like this `error: missing AMDGPU architecture for SYCL offloading; specify it with '-Xsycl-target-backend=amdgcn-amd-amdhsa --offload-arch=<gpu-arch-name>'`

Reply:
Yeah, I was wondering about that and I think it would be a good idea. I decided to keep it as it is as I wasn't sure how many places I'd need to update. But I'll give it a go and see what happens, thanks. I think I'll choose `<arch-name>` just to avoid being overly specific on "gpu", as we could be offloading to a more generic accelerator or DSP, etc

I see other diagnostics which provide `=<triple>` placeholders so it's not unprecedented.